src/avt/Database/Database/avtGenericDatabase.C

```diff
@@ -7965,15 +7965,46 @@ avtGenericDatabase::CommunicateGhostZonesFromDomainBoundariesFromFile(
 //  Creation:   August 13, 2025
 //
 //  Modifications:
+//    Kathleen Biagas, Thu Aug 21, 2025
+//    Add optional 'isPrimary' arg to ExchangeVariable to aid in setting
+//    'active' variables (eg ActiveScalars, ActiveVectors, etc).
+//    Also added help function convert between avtVarType and
+//    vtkDataSetAttribute type.
+//
 // ****************************************************************************
+
+
+int
+avtVarTypeToVTKAttributeType(avtVarType v)
+{
+    // only concerned with main vtk types
+    switch (v)
+    {
+      case AVT_SCALAR_VAR:
+        return vtkDataSetAttributes::SCALARS;
+      case AVT_VECTOR_VAR:
+        return vtkDataSetAttributes::VECTORS;
+      case AVT_TENSOR_VAR:
+        return vtkDataSetAttributes::TENSORS;
+      case AVT_SYMMETRIC_TENSOR_VAR:
+        return vtkDataSetAttributes::TENSORS;
```

Comment:
is there not a special one for symmetric tensors?

Reply:
VTK does not have a separate attribute type for symmetric tensors.